core/modules/installer/installer.browser.inc

```diff
@@ -22,11 +22,11 @@ function _installer_browser_is_project_enabled($type, $name) {
       break;
     case 'theme':
       $themes = list_themes();
-      return isset($themes[$name]);
+      return (isset($themes[$name]) && $themes[$name]->status == 1);
       break;
     case 'layout':
-      $layout = layout_load($name);
-      return !empty($layout);
+      $excluded_templates = config_get('layout.settings', 'excluded_templates');
+      return (!empty(layout_get_layout_info($name)) && !in_array($name, $excluded_templates));
```

Comment:
It also looks like `layout_get_layout_info()` is deprecated and replaced by `layout_get_layout_template_info()`.